pkg/util/config_util.go

```diff
@@ -49,6 +50,24 @@ func GetFromFile(c interface{}, filename string) error {
 	return nil
 }
 
+//WriteToJSONFile writes a struct to json file
+func WriteToJSONFile(c interface{}, filename string) error {
+	data, err := json.Marshal(c)
+	if err != nil {
+		return fmt.Errorf("unable to marshal odo config data: %w", err)
```

Comment:
this function is generic, it does not specifically write config data. Please adapt the error text

Reply:
ok